CONTRIBUTING.md

```diff
@@ -4,48 +4,120 @@ Not convinced you want to learn Clojure? Read this developer's [first month expe
 
 No Clojure or programming experience? No worries. Read this [guide](https://www.notion.so/athensresearch/Onboarding-for-New-Clojurians-b34b38f30902448cae68afffa02425c1), join our Discord, and we'll find you a Clojure learning partner.
 
-Issues tagged "[good first issue](https://github.com/athensresearch/athens/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)" are... good first issues.
+Issues tagged "[good first issue](https://github.com/athensresearch/athens/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)" are… good first issues. (But we haven’t catalogued any yet. Sorry!)
 
 # Development Environment
 
-These dependencies are needed to get Athens up and running. Follow the instructions in the links.
+## Getting Athens to run locally
 
-1. [java 11 and lein](https://purelyfunctional.tv/guide/how-to-install-clojure/) (lein installs Clojure)
-1. [node 12](https://nodejs.org/en/download/) and [yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
+These dependencies are needed to get Athens up and running. To install them, follow the instructions in the links.
+
+1. [Java 11 and Leiningen](https://purelyfunctional.tv/guide/how-to-install-clojure/) (Leiningen installs Clojure)
+1. [Node 12](https://nodejs.org/en/download/) and [Yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
+
+After you've got these dependencies, clone the Git repository to your hard drive:
 
-After you've got these dependencies, clone the git repository to your harddrive:
 ```
 git clone https://github.com/athensresearch/athens.git
 ```
+
 Then `cd athens/` and run the following commands.
 
-Pull javascript dependencies:
+Pull JavaScript dependencies:
+
 ```
 yarn
 ```
 
-Pull java dependencies and build with:
+Pull Java dependencies and build, then start a local HTTP server for Athens:
+
 ```
 lein dev
 ```
 
-When these scripts are done, your terminal will read `build complete`. Athens can be accessed by pointing a browser to `localhost:3000` on UNIX or `127.0.0.1:3000` on Windows.
+When these scripts are done, your terminal will read `build complete`. Athens can then be accessed by pointing a browser to http://localhost:3000/ on UNIX or http://127.0.0.1:3000/ on Windows.
+
+## Viewing devcards
+
+[Devcards](https://github.com/bhauman/devcards) are pages that show just one component of the web app, for the purpose of demonstrating or testing how that component looks when rendered with certain data.
+
+To open this project’s devcards:
+
+1.  Instead of `lein dev`, start a local server like this:
+
+    ```
+    lein devcards
+    ```
```

Comment:
You can also view devcards when you run `lein dev`. I realized this later after Jeff said that. The url below is correct. When you run `lein dev` and `lein devcards` at the same time, the url will actually be different, e.g. 3001. So maybe best to not mention `lein devcards` for newbees